module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraUser.md

```diff
@@ -139,6 +139,24 @@ Mark Adams bbbbbbbb-1111-2222-3333-cccccccccccc Adams@contoso.com Adams@contoso.
 
 In this example, we retrieve all users whose MailNickname starts with Ada.
 
+### Example 6: Get SignInActivity of a User
+
+```powershell
+Connect-Entra -Scopes 'User.Read.All','AuditLog.Read.All'
+Get-EntraUser -Top 1  -Property 'SignInActivity' | Select-Object -ExpandProperty 'SignInActivity'
+```
+
+```output
+lastNonInteractiveSignInRequestId : bbbbbbbb-1111-2222-3333-aaaaaaaaaaaa 
+lastNonInteractiveSignInDateTime  : 7/31/2024 1:20:28 PM
+lastSuccessfulSignInRequestId     : bbbbbbbb-1111-2222-3333-cccccccccccc 
+lastSignInDateTime                : 7/31/2024 8:18:35 AM
+lastSignInRequestId               : bbbbbbbb-1111-2222-3333-dddddddddddd 
+lastSuccessfulSignInDateTime      : 7/31/2024 1:20:28 PM
+```
+
+This example demonstrates how to retrieve the SignInActivity of a specific user by selecting a property.
```

Comment:
`-Top 1` used in a code example doesn't represent "a specific user" used in the description. `-ObjectId` (or `-UserId`, when you add support for it) would be better.